packages/components/src/Button/index.ts

```diff
@@ -0,0 +1,4 @@
+import { Button, Props, Size, Variant, variantMap } from './Button';
+
+export type { Props, Size, Variant, variantMap };
+export default Button;
```

Comment:
why not export * from ... ? 
And honestly I prefer named exports instead of default one

Reply:
You can still use named exports on components. My intention was to be able to import Button either this way

import { Button } from '@a/components'

or this way

import Button from '@a/components/Button'

But it is still in progress to achieve that. MUI has that system of exporting

Reply:
oooo good idea now i get it